tests/integration/submission/test_versioning.py

```diff
@@ -0,0 +1,280 @@
+import json
+import os
+
+from gdcdatamodel import models as md
+
+from tests.integration.submission.test_endpoints import (
+    post_example_entities_together,
+    put_example_entities_together,
+    DATA_DIR,
+)
+from tests.integration.submission.utils import data_fnames
+
+
+def release_indexd_doc(pg_driver, indexd_client, latest_did):
```

Comment:
this looks like something that can be reused elsewhere, put it in `utils.py` mb?